File: module_find/src/main/java/debug/FindApplication.java
package debug;

import android.content.Intent;

import com.ebook.api.RetrofitManager;
import com.ebook.common.BaseApplication;
import com.ebook.db.GreenDaoManager;


public class FindApplication extends BaseApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        RetrofitManager.init(this);
        GreenDaoManager.init(this);
    }
}

File: module_find/src/main/test/debug/FindApplication.kt
package debug

import com.ebook.common.BaseApplication
import com.ebook.common.event.KeyCode
import com.ebook.db.ObjectBoxManager
import com.therouter.router.addPathReplaceInterceptor
import com.therouter.router.interceptor.PathReplaceInterceptor
import dagger.hilt.android.HiltAndroidApp

@HiltAndroidApp
class FindApplication : BaseApplication() {
    override fun onCreate() {
        super.onCreate()
        ObjectBoxManager.init(context)
        addPathReplaceInterceptor(object : PathReplaceInterceptor() {
            override fun replace(path: String?): String? {
                if (path == KeyCode.Book.DETAIL_PATH) {
                    return KeyCode.Find.TEST_DETAIL_PATH
                }
                return path
            }
        })
    }
}
